web-next/app/agent/multi-agent-orchestration-best-practices/page.module.css

```
@@ -0,0 +1,559 @@
+
+@media (max-width: 980px) {
+  .sidebar {
+    display: none;
+  }
+  .content {
+    padding: 36px 22px 100px;
+  }
+}
+
+/* ---------- Mermaid Pie Chart Vivid Colors Overrides ---------- */
+.diagramWrap :global(svg path.pieCircle:nth-of-type(1)),
```

Stylelint (17.14.1)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)